src/cloudflare/sockets.ts

```diff
@@ -10,3 +10,11 @@ export function connect(
 ): sockets.Socket {
   return sockets.connect(address, options);
 }
+
+interface Fetcher {
+  fetch: typeof fetch;
+}
+
+export function createHttpClient(socket: sockets.Socket): Fetcher {
+  return sockets.createHttpClient(socket);
```

Comment:
I can imagine that this might get a bit confusing *if* we ever do decide to implement `node:http`. I wonder if an alternative name like `getFetcher(...)` might be clearer as a way to indicate that the returned client implements the `fetch(...)` api.

Reply:
Sure, I don't mind the naming, the getFetcher name seemed overloaded but it has also crossed my mind.
I don't really think it would be confusing if we decide to implement `node:http` though as this is intended to stay as an experimental only API in the near future.